code/__defines/access_defs.dm

```diff
@@ -0,0 +1,193 @@
+#define ACCESS_CONTAINMENT_SCP_078 "ACCESS_CONTAINMENT_SCP_078"
+#define ACCESS_CONTAINMENT_SCP_079 "ACCESS_CONTAINMENT_SCP_079"
+#define ACCESS_CONTAINMENT_SCP_080 "ACCESS_CONTAINMENT_SCP_080"
+#define ACCESS_CONTAINMENT_SCP_082 "ACCESS_CONTAINMENT_SCP_082"
+#define ACCESS_CONTAINMENT_SCP_087_B "ACCESS_CONTAINMENT_SCP_087_B"
+#define ACCESS_CONTAINMENT_SCP_096 "ACCESS_CONTAINMENT_SCP_096"
+#define ACCESS_CONTAINMENT_SCP_106 "ACCESS_CONTAINMENT_SCP_106"
+#define ACCESS_CONTAINMENT_SCP_113 "ACCESS_CONTAINMENT_SCP_113"
+#define ACCESS_CONTAINMENT_SCP_131 "ACCESS_CONTAINMENT_SCP_131"
+#define ACCESS_CONTAINMENT_SCP_151 "ACCESS_CONTAINMENT_SCP_151"
+#define ACCESS_CONTAINMENT_SCP_173 "ACCESS_CONTAINMENT_SCP_173"
+#define ACCESS_CONTAINMENT_SCP_247 "ACCESS_CONTAINMENT_SCP_247"
+#define ACCESS_CONTAINMENT_SCP_263 "ACCESS_CONTAINMENT_SCP_263"
+#define ACCESS_CONTAINMENT_SCP_280 "ACCESS_CONTAINMENT_SCP_280"
+#define ACCESS_CONTAINMENT_SCP_294 "ACCESS_CONTAINMENT_SCP_294"
+#define ACCESS_CONTAINMENT_SCP_343 "ACCESS_CONTAINMENT_SCP_343"
+#define ACCESS_CONTAINMENT_SCP_399 "ACCESS_CONTAINMENT_SCP_399"
+#define ACCESS_CONTAINMENT_SCP_409 "ACCESS_CONTAINMENT_SCP_409"
+#define ACCESS_CONTAINMENT_SCP_513 "ACCESS_CONTAINMENT_SCP_513"
+#define ACCESS_CONTAINMENT_SCP_529 "ACCESS_CONTAINMENT_SCP_529"
+#define ACCESS_CONTAINMENT_SCP_714 "ACCESS_CONTAINMENT_SCP_714"
+#define ACCESS_CONTAINMENT_SCP_896 "ACCESS_CONTAINMENT_SCP_896"
+#define ACCESS_CONTAINMENT_SCP_953 "ACCESS_CONTAINMENT_SCP_953"
+#define ACCESS_CONTAINMENT_SCP_966 "ACCESS_CONTAINMENT_SCP_966"
+#define ACCESS_CONTAINMENT_SCP_999 "ACCESS_CONTAINMENT_SCP_999"
+#define ACCESS_CONTAINMENT_SCP_1102_RU "ACCESS_CONTAINMENT_SCP_1102_RU"
+#define ACCESS_CONTAINMENT_SCP_1025 "ACCESS_CONTAINMENT_SCP_1025"
+#define ACCESS_CONTAINMENT_SCP_1366 "ACCESS_CONTAINMENT_SCP_1366"
+#define ACCESS_CONTAINMENT_SCP_1499 "ACCESS_CONTAINMENT_SCP_1499"
+
+
+//More normal access defines.
+#define ACCESS_DCLASS_KITCHEN "ACCESS_DCLASS_KITCHEN"
+#define ACCESS_DCLASS_BOTANY "ACCESS_DCLASS_BOTANY"
+#define ACCESS_DCLASS_MINING "ACCESS_DCLASS_MINING"
+#define ACCESS_DCLASS_JANITORIAL "ACCESS_DCLASS_JANITORIAL"
+
+#define ACCESS_KEYAUTH "ACCESS_KEYCARD_AUTH"
+#define ACCESS_ENG_COMMS "ACCESS_COMMS_ENGINEERING"
+#define ACCESS_SCI_COMMS "ACCESS_COMMS_SCIENCE"
+#define ACCESS_SEC_COMMS "ACCESS_COMMS_SECURITY"
+#define ACCESS_LOG_COMMS "ACCESS_COMMS_LOGISTICS"
+#define ACCESS_CIV_COMMS "ACCESS_COMMS_CIVILIAN"
+#define ACCESS_MED_COMMS "ACCESS_COMMS_MEDICAL"
+#define ACCESS_COM_COMMS "ACCESS_COMMS_COMMAND"
+
+#define ACCESS_SECURITY "ACCESS_SECURITY" //1
+#define ACCESS_BRIG "ACCESS_BRIG" // Brig timers and permabrig 2
+#define ACCESS_ARMORY "ACCESS_ARMORY" //3
+#define ACCESS_FORENSICS_LOCKERS "ACCESS_FORENSICS" //4
+#define ACCESS_MEDICAL "ACCESS_MEDICAL" //5
+#define ACCESS_MORGUE "ACCESS_MORGUE" //6
+#define ACCESS_TOX "ACCESS_TOXINS" //7
+#define ACCESS_TOX_STORAGE "ACCESS_TOX_STORAGE" //8
```

Comment:
Are you certain these numbers are correct?

Reply:
i dont know why we have them all accesses are strings anyway
i used regex to mass produce the #defines